assets/demo_maps/test.site.json

```diff
@@ -1013,6 +1013,7 @@
   },
   "robots": {
     "52": {
+      "fleet": "HospitalRobot",
```

Comment:
Should we add tasks to this file so we make it part of our CI pipeline? It can be done in a followup if you prefer